sample-apps/mongodb/README.md

```diff
@@ -0,0 +1,62 @@
+# MongoDB Sample App
+
+This sample application creates an Ubuntu VM integrated with Alloy for metric and log collection. This sample app utilizes cloud-init and Make commands to facilitate the setup, configuration, and monitoring of MongoDB 8.0 using the Percona mongodb_exporter embedded within Alloy.
+
+## Prerequisites
+
+Before you begin, ensure you have the following installed:
+
+- [Multipass](https://multipass.run/)
+- Docker (for rendering the cloud-init configuration)
+- Git (for cloning the repository)
+
+## Quick Start for New Users
+
+To get started with the sample app, follow these steps:
+
+1. **Clone the Repository**: 
+   ```sh
+   git clone https://github.com/grafana/integration-sample-apps.git
+   cd integration-sample-apps/mongodb
+   ```
+
+2. **Set Up Default Config**: 
+   Execute `make defaultconfig` to create a template file with default configuration variables. Modify `jinja/variables/cloud-init.yaml` to connect Alloy to an external Prometheus compatible TSDB and/or Loki server.
+
+3. **Render Cloud-init Configuration**: 
+   Run `make render-config` to generate the `cloud-init.yaml` file based on your configuration.
+
+4. **Create and Set Up VMs**: 
+   Use `make run` to start the MongoDB sample app.
+
+6. **Stop and Clean Up**: 
+   Use `make stop` to clean up the VM and `make clean` to remove temporary files.
+
+## Make Commands
+
+- `make defaultconfig`: Initializes the configuration file with default values for cloud-init templates.
```

Comment:
Would it be helpful to mention what the default setup would be?

e.g single node, sharded cluster?

Reply:
Yes good point, will update this!